src/main/java/org/maxgamer/rs/command/commands/Warp.java

```diff
@@ -45,6 +45,7 @@ public Destination findByPrefix(String name) {
 
 	public Warp(){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)